### PR TITLE
Updated Delete command, Activity Test Class, WanderlustDateTime

### PR DESCRIPTION
**Pull request by jiaweiteo**
Wont pass CI cause i changed the command class to take in travelplanner.modelManager, so most commands don't recognise it now. Waiting for Tim and Hai Shan to deconflict

### CODE REVIEW

**Review by jeannetoh99 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/wanderlustlogic/Logic.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     * Executes the command and returns the result.
+     * @param commandText The command as entered by the user.
+     * @return the result of the command execution.
+     * @throws CommandException If an error occurs during command execution.
+     * @throws ParseException If an error occurs during parsing.
+     */
+    CommandResult execute(String commandText) throws CommandException, ParseException;
+
+    /**
+     * Returns the Travelplan.
```
Comment (jeannetoh99) [inline src/main/java/seedu/address/logic/wanderlustlogic/Logic.java:27]: I think this should be return the Travelplanner?

File: src/main/java/seedu/address/model/commons/Name.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
      * The first character of the Name must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "(\\p{Graph}+\\p{Space}?)+";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```
Comment (jeannetoh99) [inline src/main/java/seedu/address/model/commons/Name.java:19]: What is the reason for this change? Previously I changed it to Graph so that it allows for names with some punctuations, such as "Alice's Home", i also tested it so it was working.

File: src/main/java/seedu/address/logic/wanderlustlogic/Logic.java (hunk 1 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/logic/wanderlustlogic/Logic.java:27]: Okay changed!

File: src/main/java/seedu/address/model/commons/Name.java (hunk 2 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/model/commons/Name.java:19]: This test will fail so i changed it back.
image

Comment (jeannetoh99) [reply src/main/java/seedu/address/model/commons/Name.java:19]: I see, then can change to `"(\\p{Alnum}p{Graph}*\\p{Space}?)+"` ?

**Review by jeannetoh99 — APPROVED** [review]
LGTM
